### PR TITLE
[SDPA-5151] remove featured_news widget option

### PR DESCRIPTION
**Pull request by vincent-gao**
### Jira
https://digital-engagement.atlassian.net/browse/SDPA-5151

### Changes
1. Adds an update hook to remove featured_news option that appears on node form widget.

https://github.com/dpc-sdp/tide_landing_page/pull/142

### CODE REVIEW

File: tide_publication.install [hunk 1]
```diff
@@ -664,3 +664,23 @@ function tide_publication_update_8011() {
...
+ */
+function tide_publication_update_8012() {
+  $field = FieldConfig::loadByName('node', 'publication_page', 'field_landing_page_component');
+  if ($field) {
+    $handler_settings = $field->getSetting('handler_settings');
+    if (isset($handler_settings['target_bundles'])) {
+      if (isset($handler_settings['target_bundles']['featured_news'])) {
+        unset($handler_settings['target_bundles']['featured_news']);
+      }
+      if (isset($handler_settings['target_bundles']['featured_news'])) {
```
Comment (yeniatencio) [inline tide_publication.install]: This condition looks the same like you have in #L676.

**Review by anthony-malkoun — CHANGES_REQUESTED** [review]
Do the tests need updating now?

File: tide_publication.install (hunk 1 above)
Comment (vincent-gao) [reply tide_publication.install]: oh~ Thanks, changed.

**Comment by vincent-gao** [comment]
> Do the tests need updating now?

Thanks, changed.

**Review by MdNadimHossain — APPROVED** [review]

**Pull request merged**